src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs

```diff
@@ -18,6 +21,9 @@
 [assembly: LoadableClass(CopyColumnsTransform.Summary, typeof(CopyColumnsTransform), null, typeof(SignatureLoadDataTransform),
```

Comment:
>LoadableClass [](start = 11, length = 13)

remove altogether

Reply:
with `SignatureLoadRowMapper` as a signature. 

---
In reply to: [211794398](https://github.com/dotnet/machinelearning/pull/706#discussion_r211794398) [](ancestors = 211794398)